models/index.js

```diff
@@ -17,20 +20,34 @@ const {
   DB: dbTitle,
   DB_USER: dbUser,
   DB_PASS: dbPass,
-  DB_HOST: dbHost,
-  JAWSDB_URL: dbJawsUrl
+  DB_HOST: dbHost
 } = process.env;
 
+// Setting options for db connections
+const options = {
+  host: dbHost,
+  port: 3306,
+  user: dbUser,
+  password: dbPass,
+  database: dbTitle,
+  // Whether or not to automatically check for and clear expired sessions:
+  clearExpired: true,
+  // How frequently expired sessions will be cleared; milliseconds:
+  checkExpirationInterval: 900000,
+  // The maximum age of a valid session; milliseconds:
+  expiration: 86400000
+};
+
 // Starting sequelize connection with env variables
-const sequelize = new Sequelize(dbJawsUrl || dbTitle, dbUser, dbPass, {
+const sequelize = new Sequelize(dbTitle, dbUser, dbPass, {
   host: dbHost,
   dialect: 'mysql'
 });
 
-// const sequelize = config.use_env_variable
-//   ? new Sequelize(process.env[config.use_env_variable])
-//   : new Sequelize(config.database, config.username, config.password, config);
+// Starting session storage
+const mysqlStore = new MySQLStore(options);
```

Comment:
How is session data consumed in passport?

Reply:
Passport accesses the data column in the sessions table created by MySQLStore, added a comment to explain this